.github/workflows/canary-ubuntu.yml

```diff
@@ -0,0 +1,40 @@
+name: Canary-Ubuntu
+
+on:
+  schedule:
+    - cron:  '0 10 * * *'
+  workflow_dispatch:
+
+jobs:
+  canary-test-ubuntu:
+    if: github.repository == 'aws-samples/djl-demo'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04]
```

Comment:
do we want to test centos7 with old glibc here as well

Reply:
centos need use docker container